docs/getting_started.rst

```diff
@@ -7,9 +7,16 @@ Getting Started
 Features
 ========
 
-**niveristand** allows you to:
+**niveristand** has two major capabilities: system definition and real-time sequence scripting. It is recommended that you use an editor with code completion, such as `Visual Studio Code <https://code.visualstudio.com/docs/languages/python/>`_, to make it easier to browse and use this code.
```

Comment:
You might get a technical writer to take a pass at this.

Reply:
I've updated this based on the tech writer's comments.